lua/packer.lua

```diff
@@ -99,8 +107,9 @@ packer.init = function(user_config)
     vim.cmd [[command! PackerUpdate   lua require('packer').update()]]
     vim.cmd [[command! PackerSync     lua require('packer').sync()]]
     vim.cmd [[command! PackerClean    lua require('packer').clean()]]
-    vim.cmd [[command! PackerCompile  lua require('packer').compile()]]
+    vim.cmd [[command! -nargs=* PackerCompile  lua require('packer').compile(<q-args>)]]
```

Comment:
I haven't tested this yet, but we could maybe get rid of `parse_args` by using `f-args` rather than `q-args` here? I'm not certain that will work because we're passing the args to a `lua` expression rather than calling a vimscript function.